### DISCUSSION TITLE
Discord server?

### DISCUSSION THREAD

**Discussion by tabbyisntacat** [Fr0go1/Aeonix-Cs2#11] [Ideas]
A simple discord server for chatting sharing configs etc.

**Comment by Fr0go1** [comment]
Ok, i will make one this arvo

**Reply by tabbyisntacat** [reply]
Awsome, Put the link in the readme or something so i can find it.

**Reply by Fr0go1** [reply]
https://discord.gg/2b66kqG2nK

**Reply by Fr0go1** [reply]
its a old one from when i made scripts but it should work
